spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/store/jwk/JwkTokenStore.java

```diff
@@ -86,16 +88,23 @@
  * @author Joe Grandja
  */
 public final class JwkTokenStore implements TokenStore {
-	private final JwtTokenStore delegate;
```

Comment:
Ok agreed there could be another Jwt specific implementation of `TokenStore`. And although the member variable is completely hidden in the class, it's always best practice to reference against the interface.